### PR TITLE
Add styling to app

### PR DESCRIPTION
**Pull request by Daniel-Kigozi**
In this milestone, I did the following
- Add styling to bookstore app

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,27 +1,366 @@
...
+  box-shadow: 0 5px 8px 0 rgba(0, 0, 0, 0.2);
+  box-sizing: border-box;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  border-bottom: solid 1px #e8e8e8;
+  box-shadow: rgba(149, 157, 165, 0.2) 0 8px 24px;
+}
```
Comment (TadesseDev) [inline src/index.css]: - [ ] dear @Daniel-Kigozi, I notice your heading has no margin/padding from the right side and the navigation items are added to the bottom of the `Book Store` text. Please make sure to add the right color for the navigations, the appropriate spacing between elements and align all the elements inside the heading as shown in the [Zeplin](https://zeplin.io/) design.
- [ ] Also, There is too much padding from the top and left side for the display. Please consider reducing it. This is to make sure your app looks exactly like the given design.
- [ ] Besides the above points, I believe
### Figma design 
Capturex

### Your design.
 you missed the part where you add a user icon on the right corner of your header.
Capture

File: src/index.css [hunk 2]
```diff
@@ -1,27 +1,366 @@
...
+.book {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 13px;
+  margin-top: 50px;
+  width: 80%;
+  border: solid 1px #e8e8e8;
+  background-color: white;
+  padding: 10px 0 50px 0;
+}
```
Comment (TadesseDev) [inline src/index.css]: - [ ] Dear @Daniel-Kigozi, Your design is looking good already, But you have not added the `Circe` icon next to the `number with percent` to represent the percentage. 
- [ ] Please, notice how elements are aligned inside the [Zeplin](https://zeplin.io/) design, The cards and the `Book Store` text are aligned equally on the left side and the `user icon` and the `books` are aligned equally on the right side. Please consider fixing this for a better layout.
- [ ] Also, Please notice there is a color difference between your design and [Zeplin](https://zeplin.io/) design on the `Book subtitle`.

### Figma design 
figma

### Your design.
your

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Daniel-Kigozi 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7

## Highlights
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitFlow  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @Daniel-Kigozi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**